src/components/sidebar/types.d.ts

```diff
@@ -0,0 +1,26 @@
+type NodeT = import("react").ReactNode
```

Comment:
Why do we use this NodeT here instead of a `ReactNode` ?

Reply:
Just to write less

Reply:
Just a minute we introduce a new type just because it has a smaller and more handy name? I think this is not correct. I would rename a lot of names with names I prefer but this would make my code much less readable. I can create e.g a CompT for ReactComponent etc because its easier to write but this would make my code not very easy to read for others I think